nodes/For.py

```diff
@@ -18,12 +18,19 @@
 
 Example:
     - target  -> AssignName(i)
-    - iter    -> Call(func = Name(range), args = [3])
+    - iter    -> List(Const.int(Value=1), Const.int(Value=2),
```

Comment:
This is an okay change, but the *real* change should be made in the documentation of `iter` above, which incorrectly says that this must be a `Call` node. Really it's just a node that represents the value being iterated over - the part after the `in`.